src/sentry/south_migrations/0464_auto__add_sentryappcomponent__add_field_sentryapp_schema.py

```diff
@@ -0,0 +1,1339 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding model 'SentryAppComponent'
+        db.create_table('sentry_sentryappcomponent', (
+            ('id', self.gf('sentry.db.models.fields.bounded.BoundedBigAutoField')(primary_key=True)),
+            ('uuid', self.gf('sentry.db.models.fields.uuid.UUIDField')(
+                auto_add='uuid:uuid4', unique=True, max_length=32)),
+            ('type', self.gf('django.db.models.fields.CharField')(max_length=64)),
+            ('schema', self.gf('sentry.db.models.fields.encrypted.EncryptedJsonField')(default={})),
```

Comment:
do we want to take the `default={}` out here?

Reply:
If we want it to be `null=False` we need a default and the EncryptedJSONField defaults to this.

Reply:
If it's `NOT NULL` we need this default. And since it's a JSONField, we need it to be valid JSON (an empty dict).